terraform/azure-mongodb/azure-mongodb.tf

```diff
@@ -115,6 +115,11 @@ resource "azurerm_cosmosdb_mongo_collection" "mongo-collection" {
 
 	default_ttl_seconds = "777"
 	shard_key           = var.shard_key
+
+	index {
+		keys = [var.shard_key]
+		unique = true
```

Comment:
I wonder if this should be configurable. Was this the default in previous versions?